zipkin-collector-service/config/collector-cassandra.scala

```diff
@@ -25,11 +25,18 @@ import org.twitter.zipkin.storage.cassandra.ZipkinRetryPolicy
 val contactPoints: Array[String] = sys.env.get("CASSANDRA_CONTACT_POINTS").getOrElse("localhost")
   .split(",")
 
-val cluster = Cluster.builder()
-  .addContactPoints(contactPoints:_*)
-  .withSocketOptions(new SocketOptions().setConnectTimeoutMillis(10000).setReadTimeoutMillis(20000))
-  .withRetryPolicy(ZipkinRetryPolicy.INSTANCE)
-  .build()
+val cassandraUser: String = sys.env.get("CASSANDRA_USER").getOrElse("")
+val cassandraPass: String = sys.env.get("CASSANDRA_PASS").getOrElse("")
+
+val clusterBuilder = if(cassandraUser == "" && cassandraPass == "")
+  Cluster.builder()
+else
+  Cluster.builder().withCredentials(cassandraUser, cassandraPass)
+
+val cluster = clusterBuilder.addContactPoints(contactPoints:_*)
+    .withSocketOptions(new SocketOptions().setConnectTimeoutMillis(10000).setReadTimeoutMillis(20000))
+    .withRetryPolicy(ZipkinRetryPolicy.INSTANCE)
+    .build()
```

Comment:
Not that I can see, I think you implemented the SpanStoreBuilder to be backwards compatible.  Will roll it forward.